src/Streamly/Prelude.hs

```diff
@@ -600,8 +600,8 @@ module Streamly.Prelude
     --
     -- The following equations hold for lists:
     --
-    -- > scanl f z xs == map (foldl f z) $ inits xs
-    -- > scanr f z xs == map (foldr f z) $ tails xs
+    -- prop> scanl f z xs == map (foldl f z) $ inits xs
+    -- prop> scanr f z xs == map (foldr f z) $ tails xs
```

Comment:
These examples are conceptual examples based on Data.List, we cannot doctest these for streams, if at all we want to doctest them then we should use the Data.List versions of these functions.